dipper/sources/AnimalQTLdb.py

```diff
@@ -599,14 +614,15 @@ def _process_qtls_genetic_location(
 
                     assoc.add_association_to_graph()
 
+                # off by one - the following actually gives us (limit + 1) records
                 if not self.test_mode and limit is not None and line_counter > limit:
```

Comment:
note: 
the non-generic readers   (e.g. filereader = csv.reader) often have builtin line counters
so 
```            filereader.line_num > limit
```

Reply:
Yep, this one has a built-in line counter. I'm not fixing this here in the interest of a more focused PR.